background.ts

```diff
@@ -53,4 +53,14 @@ WebExtensionBlocker.fromPrebuiltAdsAndTracking().then((blocker: WebExtensionBloc
   blocker.on('request-blocked', incrementBlockedCounter);
   blocker.on('request-redirected', incrementBlockedCounter);
   console.log('Ready to roll!');
+
+  chrome.browserAction.onClicked.addListener(() => {
+    if (blocker.isBlockingEnabled(browser)) {
+      blocker.disableBlockingInBrowser(browser);
```

Comment:
This will disable the adblocker completely, right? Should we not only disable for current tab?